dev-tools/src/lambdas/sqsOperations/handler.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { when } from 'jest-when'
+import { SqsOperation } from '../../types/sqsOperation'
+import { addMessageToQueue } from './addMessageToQueue'
+import { handler } from './handler'
+
+jest.mock('./addMessageToQueue', () => ({
+  addMessageToQueue: jest.fn()
+}))
+
+describe('handler', () => {
```

Comment:
Rather than just putting "handler", which will output something confusing to the test output, can you add something descriptive like "sqs operations handler"?

Reply:
Done